tests/impl.cpp

```diff
@@ -2858,6 +2858,19 @@ result_t test_mm_cmpeq_epi8(const SSE2NEONTestImpl &impl, uint32_t i)
                         d12, d13, d14, d15);
 }
 
+result_t test_mm_cmpeq_pd(const SSE2NEONTestImpl &impl, uint32_t i)
+{
+    const double *_a = (const double *) impl.mTestFloatPointer1;
+    const double *_b = (const double *) impl.mTestFloatPointer2;
+    uint64_t d0 = (_a[0] == _b[0]) ? 0xffffffffffffffff : 0x00;
+    uint64_t d1 = (_a[1] == _b[1]) ? 0xffffffffffffffff : 0x00;
+
+    __m128d a = do_mm_load_pd(_a);
+    __m128d b = do_mm_load_pd(_b);
+    __m128d c = _mm_cmpeq_pd(a, b);
+    return validateDouble(c, (double) d0, (double) d1);
```

Comment:
I wonder the casting is appropriate or not.
For example, if `d0` is `1`, then `(double) d0` would be `1.0`.
I think `1.0` would not be `0x0000000000000001`.